sdk/python/requirements/py3.10-ci-requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.9
```

Comment:
I think this is incorrect

Reply:
you are right, I need to create a python 3.10 venv and run the command from Makefile. let me fix this.

Reply:
fixed, let's see the testing results.

Reply:
seems integration test failed...

Reply:
21 fails,  the most frequent error is about the wrong format of Timestamp:    google.api_core.exceptions.BadRequest: 400 Error while reading data, error message: Invalid timestamp microseconds value 1700011424237000000 of logical type NONE; in column 'created' 

let me dig into it and see what's the root cause

Reply:
1, The Google's BigQuery api only accepts "ms" resolution for timestamp, while the Pyarrow.parquet.write_table() will maintain the resolution to the exact original resolution which is "ns" by default.